Source/MDFastBindingBlueprint/Private/BlueprintExtension/MDFastBindingWidgetBlueprintExtension.cpp

```diff
@@ -101,7 +91,36 @@ void UMDFastBindingWidgetBlueprintExtension::HandleEndCompilation()
 {
 	Super::HandleEndCompilation();
 
-	CompilerContext = nullptr;
+	if (CompilerContext)
+	{
+		// If this class already has the class extension serialized, it had already been saved prior
+		// There's a chance it's being freshly loaded here so we need to generate the binding graphs
+		if (UWidgetBlueprintGeneratedClass* WidgetBPClass = Cast<UWidgetBlueprintGeneratedClass>(CompilerContext->NewClass))
+		{
+			if (WidgetBPClass->GetExtension<UMDFastBindingWidgetClassExtension>() != nullptr)
+			{
+				RefreshPinnedGraphs();
+			}
+		}
+
+		CompilerContext = nullptr;
+	}
+}
+
+void UMDFastBindingWidgetBlueprintExtension::RefreshPinnedGraphs()
```

Comment:
The "Refresh" naming is a bit misleading since it only populates the Pinned Graphs for the first time it's called

Reply:
That makes sense, I'll change it to `PopulatePinnedGraphs`

Reply:
Resolved in commit [5559d6b](https://github.com/DoubleDeez/MDFastBinding/pull/40/commits/5559d6b4a030188164ffc031f228b2c46064e36b)